src/unistd.rs

```diff
@@ -2112,10 +2112,17 @@ mod setres {
     /// * `suid`: saved user id
     /// * returns: Ok or libc error code.
     ///
+    /// Pass None to denote no change
```

Comment:
Better if you can format the comment blocks in the more standard way, with Parameters and Returns sections, like `fpathconf`` does.